content/kubermatic/master/tutorials_howtos/how_to_contribute_to_kkp/_index.en.md

```diff
@@ -0,0 +1,63 @@
++++
+title = "How To Contribute to Kubermatic Kubernetes Platform (KKP)"
+date = 2021-04-24T15:00:15+02:00
+weight = 25
++++
+
+
+## Contributing to KKP
+
+Kubermatic Kubernetes Platform (KKP) is an open-source project to centrally manage the global automation of thousands of Kubernetes clusters across multi-cloud, on-prem and edge with unparalleled density and resilience. Therefore, contributors in different areas but not limited to project automation, code review, tooling, documentation, testing etc., for the holistic improvement of the KKP project is highly welcomed. Kubermatic projects are [Apache 2.0 licensed](https://github.com/kubermatic/kubermatic/blob/master/LICENSE) and accept contributions via GitHub pull requests from interested contributors. This document outlines some of the conventions on development workflow, contributor's notes, contributing steps, contact points and other resources to make your contribution to the projects felicitous.
```

Comment:
Can we not mention "KUBERMATIC KUBERNETES PLATFORM (KKP)" again and again and again? The user is already in the KKP documentation, I would say that chapters do not need to include ".... to Kubermatic Kubernetes Platform (KKP)" again. It just reads awkwardly.

Reply:
Ok. But I think it is fine with the main heading?